lib/command.py

```diff
@@ -40,6 +41,19 @@ def say(self, player: Player, message: str):
                 # send them a message telling them what the player said
                 other_player.message(f"{player.name} says: {message}")
 
+    def tell(self, player: Player, params):
+        """ tell <player> message - Whisper a private message to the target player."""
+        target = params.split(' ')[0]
+        message = ' '.join(params.split(' ')[1:])
+
+        if target.lower() not in [player.name.lower() for player in self.game_state.list_other_players(player)]:
```

Comment:
If we want to keep working on this - the `lower` thing is smart, but it's expensive to recompute on commands - we might be better indexing by lower case

Reply:
Oh, I think I understand.

Reply:
Sorry, I suppose I should say, have the key lowercase